roles/check-swlib/tasks/main.yml

```diff
@@ -16,13 +16,27 @@
 - name: check-swlib | Validate base software release
   shell: |
     set -o pipefail
-    {% for i in item.files %}
-      gsutil ls -L gs://{{ swlib_mount_src }}/{{ i }} | \
-        awk '/md5/ { print $3 }'
+    {% for f in item.files %}
+    if gsutil ls gs://{{ swlib_mount_src }}/{{ f.name }} >/dev/null 2>&1; then
+      f_name="{{ f.name }}"
+      md5sum="{{ f.md5sum }}"
+    else
+      f_name="{{ f.alt_name | default('x') }}"
+      md5sum="{{ f.alt_md5sum | default('x') }}"
+    fi
+    actual_md5=$(gsutil ls -L gs://{{ swlib_mount_src }}/${f_name} | awk '/md5/ { print $3 }')
+    if [ $? -ne 0 ] ; then
+      echo "ERROR locating {{ f.name }} {{ f.alt_name | default('') }}"
+      exit 1;
+    else
+      if [ "$actual_md5" != "$md5sum" ]; then
+        echo "ERROR locating {{ item.name }}: md5 mismatch for ${f_name}"
```

Comment:
Technically this isn't an error _locating_ the item.  Just that the file contents don't match.  I'd even put expected and actual values in the error message, so a technical user could troubleshoot based on the error message alone.